Vimeo-DotNet/VimeoAPI/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="5.0.5" targetFramework="net40" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net46" />
```

Comment:
Something had gone awry, so I removed it and reinstalled it, to get the update in the PROJ file, too. Fixed in 081eb73.